docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -115,3 +115,14 @@ instances, which can be provided using any JSON marshalling library (such as Cir
 
 When implementing a custom support trait, one should simply extend the `EntityStreamingSupport` abstract class,
 and implement all of it's methods. It's best to use the existing implementations as a guideline.
+
+## Supporting custom content types
+
+In order to marshal into custom content types both a `Marshaller` that can handle that content type, 
+**as well as an `EntityStreamingSupport` of matching content type** is required.
```

Comment:
This PR is addressing a bug / problem we have in the existing API - people getting `[]` where they expected a proper stream of elements. The sole purpose of this PR is to help those avoid this pitfall, I'm totally open to looking at the ts-reaktive project, however this is unrelated to this PR which attempts to patch up status quo.

We can eventually deprecate the current one, and replace with a better one - if that better one is ts-reaktive then cool. Let's discuss that separately.